packages/turf-line-arc/index.ts

```diff
@@ -57,21 +57,19 @@ export default function lineArc(
   const arcStartDegree = angle1;
   const arcEndDegree = angle1 < angle2 ? angle2 : angle2 + 360;
 
-  let alfa = arcStartDegree;
+  let alpha = arcStartDegree;
   const coordinates = [];
   let i = 0;
-
-  while (alfa < arcEndDegree) {
+  // How many degrees we'll swing around between each step.
+  const arcStep = (arcEndDegree - arcStartDegree) / steps;
+  // Add coords to the list, increasing the angle from our start bearing
+  // (alpha) by arcStep degrees until we reach the end bearing.
+  while (alpha <= arcEndDegree) {
```

Comment:
I wonder if this gets weird with floating point logic with N steps and very occasionally skips the final step.

Reply:
Had crossed my mind. I did some simple testing and if there are precision errors in the reverse direction they tend to go low:

```
let steps = 1234567890; // Tried various values for steps
const x = 1 / steps;
// x 8.10000007371e-10

// Re-multiplication never seems to *exceed* the original value
const t = 0 + x * steps;
// t 1
// t 0.9999999985906001
// t 0.9999999978858999
```
If the bearing tends a little short then the <= should let that loop get as close as possible to the endBearing without going a whole step over.

We could go back to a "calculate all the step angles except the last one and use the endAngle for the last" approach. From what I can see above though JS floating point shoudl be ok, and the code is simpler.

Reply:
With a slightly different construction I think we can go over, but I appreciate you digging in. I feel good about this and we can always substitute the arcEndDegree for the final point later if there's something we didn't forsee.

```
let steps = 1e5;
const x = 1/steps;
let v = 0;
for (let i=0; i<steps; i++) {
    v+=x;
}
v
0.9999999999980838


let steps = 1e10; // warning this is pretty slow
const x = 1/steps;
let v = 0;
for (let i=0; i<steps; i++) {
    v+=x;
}
v
1.0000000694756235
```

Reply:
Sounds good.